### DISCUSSION TITLE
Call for videos

### DISCUSSION THREAD

**Discussion by otseng** [eliranwong/UniqueBible#804] [Show and tell]
I'd like to create a wiki page of links to YouTube videos on how to use UBA.  

Can you create a 5 min video on YouTube on how to do the following?  Post it here and I'll then put it up on the wiki.

* How to install and run UBA
* How to configure UBA
* How to use the toolbars
* How to configure the GUI layout
* How to download resources
* How to select a Bible
* How to use master and mini controls
* How to use UBA in another language
* How to search
* How to use notes
* How to use live filter
* How to use the library catalog
* How to use Bible collections
* How to customize a Bible font
* How to use lexicons
* How to download a video from YouTube and watch it
* How to listen to the Bible
* How to use devotionals
* How to create your own devotional
* How to use commentaries
* How to study original languages
* How to do morphology search
* How to do highlighting
* How to create your own book
* How to read PDF files
* How to read ePUB files
* How to display hymns
* How to use commands
* How to use the Bible reading plan
* How to use plugins
* How to write plugins
* How to use macros
* How to import third party modules
* How to use cross references
* How to create your own cross reference file
* How to use UBA as a webserver and run a website
* How to use UBA as a telnet server
* How to use remote CLI
* How to use UBA to present verses and songs during a church service
* How to run on Android device as a webserver
* How to share content with other mobile phones from your Android device
* How to create your own Bible
* How to use discourse analysis
* etc

**Comment by ghost** [comment]
I take it the videos will be reviewed. This is becoming an awesome tool.

**Reply by otseng** [reply]
Yes, videos will be reviewed.  If approved, will post on wiki.  (For now, the bar for approval is quite low)

**Comment by otseng** [comment]
I've created the first video and posted it here:
https://github.com/eliranwong/UniqueBible/wiki/Video-Tutorials
